packages/dialtone-vue2/components/filter_pill/filter_pill.vue

```diff
@@ -0,0 +1,322 @@
+    },
+
+    /**
+     * Text shown in tooltip when you hover the alpha button,
+     * required if no content is passed to default slot
+     */
+    alphaTooltipText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Sets the element to which the
+     * <a class="d-link" href="https://dialtone.dialpad.com/components/popover.html#vue-api" target="_blank">popover component</a>
+     *  is going to append to
+     * @values body, parent, root, HTMLElement
+     */
+    appendTo: {
+      type: [HTMLElement, String],
+      default: 'body',
+      validator: appendTo => {
+        return POPOVER_APPEND_TO_VALUES.includes(appendTo) ||
+          (appendTo instanceof HTMLElement);
+      },
+    },
+
+    /**
+     * HTML disabled attribute
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * If the popover does not fit in the direction described by "placement",
+     * it will attempt to change its direction to the "fallbackPlacements".
+     * <a class="d-link" href="https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements" target="_blank">Popper.js docs</a>
+     * */
+    fallbackPlacements: {
+      type: Array,
+      default: () => ['top-start', 'auto'],
+    },
+
+    /**
+     * Label of the button
+     */
+    label: {
+      type: String,
+      default: undefined,
+    },
+
+    /**
+     * Determines maximum height for the popover before overflow.
+     * Possible units rem|px|em
+     */
+    maxHeight: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Determines maximum width for the popover before overflow.
+     * Possible units rem|px|%|em
+     */
+    maxWidth: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Text shown in tooltip when you hover the omega button,
+     * required as it is an icon only button
+     */
+    omegaTooltipText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Padding size class for the popover content.
+     * @values none, small, medium, large
+     */
+    padding: {
+      type: String,
+      default: 'large',
+      validator: (padding) => {
+        return Object.keys(POPOVER_PADDING_CLASSES).some((item) => item === padding);
+      },
+    },
+
+    /**
+     * The direction the popover displays relative to the anchor.
+     * <a class="d-link" href="https://atomiks.github.io/tippyjs/v6/all-props/#placement" target="_blank">Tippy.js docs</a>
+     * @values top, top-start, top-end, right, right-start, right-end, left, left-start, left-end, bottom, bottom-start, bottom-end, auto, auto-start, auto-end
+     */
+    placement: {
+      type: String,
+      default: 'bottom-start',
+    },
+
+    /**
+     * Shows the clear button
+     */
+    showClear: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * The size of the button.
+     * @values xs, sm, md, lg, xl
+     */
+    size: {
+      type: String,
+      default: 'md',
```

Comment:
```suggestion
      default: 'sm',
```

I'd say nearly all Filter Pill uses will be this size.